core/blockly.js

```diff
@@ -63,37 +64,41 @@ goog.requireType('Blockly.Workspace');
  * compiler to override this constant.
  * @define {string}
  */
-Blockly.VERSION = 'uncompiled';
+exports.VERSION = 'uncompiled';
 
 // Add a getter and setter pair for Blockly.mainWorkspace, for legacy reasons.
-Object.defineProperty(Blockly, 'mainWorkspace', {
-  set: function(x) {
-    Blockly.utils.deprecation.warn(
-        'Blockly.mainWorkspace', 'September 2021', 'September 2022');
-    Blockly.common.setMainWorkspace(x);
-  },
-  get: function() {
-    Blockly.utils.deprecation.warn(
-        'Blockly.mainWorkspace', 'September 2021', 'September 2022',
-        'Blockly.getMainWorkspace()');
-    return Blockly.common.getMainWorkspace();
+Object.defineProperties(exports, {
+  mainWorkspace: {
+    set: function(x) {
+      Blockly.utils.deprecation.warn(
+          'Blockly.mainWorkspace', 'September 2021', 'September 2022');
+      Blockly.common.setMainWorkspace(x);
+    },
+    get: function() {
+      Blockly.utils.deprecation.warn(
+          'Blockly.mainWorkspace', 'September 2021', 'September 2022',
+          'Blockly.getMainWorkspace()');
+      return Blockly.common.getMainWorkspace();
+    }
   }
 });
 
 // Add a getter and setter pair for Blockly.selected, for legacy reasons.
-Object.defineProperty(Blockly, 'selected', {
-  get: function() {
-    Blockly.utils.deprecation.warn(
-      'Blockly.selected', 'September 2021', 'September 2022',
-      'Blockly.common.getSelected()');
-    return Blockly.common.getSelected();
-  },
-  set: function(newSelection) {
-    Blockly.utils.deprecation.warn(
-      'Blockly.selected', 'September 2021', 'September 2022',
-      'Blockly.common.setSelected()');
-    Blockly.common.setSelected(newSelection);
-  }
+Object.defineProperties(exports, {
```

Comment:
Since `defineProperties` can do more than one, this could be combined with the previous into a single call.  (And another similar case below.)

Reply:
Done